en/integrations/plugin-packs/tutorials/azure-credential-configuration.md

```diff
@@ -0,0 +1,100 @@
+Azure CLI method is not be compatible with host autodiscovery.
+
+## Azure API set up 
+
+For Centreon to be able to request information about Azure resources through **Azure API**, 4 tokens are needed:
+* **subscription ID**
+* **tenant ID**
+* **client ID**
+* **client secret** 
+
+To have these tokens, you need to create a new application in Azure. The below steps detail how to create this application and set the right permissions to monitor Azure resources.
+
+* Create an *Application* in Azure Active Directory:
+	- Log in to Azure Portal.
+	- Select **Azure Active Directory**.
+	- On the left menu, go to **App registrations**.
+	- Click on **+ New registration**, put the name of your application and then click on **Register**.
+	
+* Add the **Monitoring Reader role** to your new application:
+	- Go to **Resource groups**.
+	- Choose the resource group containing the resources you want to monitor. 
+	- Click on **Access Control (IAM)** and **+ Add > Add role assignment**.
+	- Search for **Monitoring Reader role** and click Next.
+	- Select your new application as a member for this role by clicking on **+ Select members**.
+	- Review and assign.
+	
+Now that the application is created, the below steps will guide you to retrieve the 4 tokens: 
+* Get the **subscription ID**:
+	- Go to **Subscriptions** in Azure portal. 
+	- Next to the name of your subscription you have the **Subscription ID**.
+* Get the **Tenant ID**:
+	- Go to **Azure Active Directory**.
+	- In **Overview**, under the name of the directory you will see the **Tenant ID**.
+	
+* Get the **Client ID**:
+	- Go to **Azure Active Directory**.
+	- On the left, select **App registrations** and the name of your application. 
+	- In the **Overview** panel you will have **Application (client) ID**.
+* Get the **Client secret**:
+	- Still in **Azure Active Directory > App registrations**, go to **Certificates & secrets**.
+	- Click on **+ New client secret**, set description as well as expiration date and Add.
+	- Save the **key value**, it is the **client secret** and you won't be able to retrieve it after you leave this page.
+	
+These 4 tokens will be mandatory when using host autodiscovery and Centreon Azure based plugins. 
+
+## Azure CLI set up 
+
+*These steps are not required if you can use Azure API.* 
+
+Install Azure CLI:
+
+```bash
+sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+sudo echo -e "[azure-cli]\nname=Azure CLI\nbaseurl=https://packages.microsoft.com/yumrepos/azure-cli\nenabled=1\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" > /etc/yum.repos.d/azure-cli.repo
+sudo yum install azure-cli-2.29.2-1.el7
+```
+
+Then, use the **centreon-engine** account to obtain a token:
+
+```bash
+su - centreon-engine
+az login
+```
+
+The shell will output this message including an authentication code:
+
+	*To sign in, use a web browser to open the page https://microsoft.com/devicelogin*
+	*and enter the code CWT4WQZAD to authenticate.*
```

Comment:
```suggestion
```text
To sign in, use a web browser to open the page https://microsoft.com/devicelogin
and enter the code CWT4WQZAD to authenticate.*
```
```